hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/TestOMClientRequestWithUserInfo.java

```diff
@@ -95,7 +96,8 @@ public void testUserInfo() throws Exception {
     String volumeName = UUID.randomUUID().toString();
     OzoneManagerProtocolProtos.OMRequest omRequest =
         OMRequestTestUtils.createBucketRequest(bucketName, volumeName, true,
-            OzoneManagerProtocolProtos.StorageTypeProto.DISK);
+            OzoneManagerProtocolProtos.StorageTypeProto.DISK,
+            ClientVersion.DEFAULT_VERSION.toProtoValue());
```

Comment:
If a version needs to be specified, the latest version should be used, not the earliest, unless this is doing client cross compat testing, which I don't think it is. That would be `ClientVersion.CURRENT`

Reply:
Why is this change concerned with client version?

Reply:
@errose28 All the existing tests were using `ClientVersion` 0 to execute [this block](https://github.com/xBis7/ozone/blob/master/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java#L275-L284). In our case we needed to test the [else](https://github.com/xBis7/ozone/blob/master/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java#L285-L298) statement that needs to use the latest `ClientVersion` in order to execute. For consistency and not making any changes to the existing tests, we used `ClientVersion.DEFAULT_VERSION`. Also see this [comment here](https://github.com/apache/ozone/pull/3750#discussion_r971350019) as @neils-dev mentioned.